presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -607,10 +607,10 @@ public StatsCalculator getStatsCalculator()
     }
 
     @Override
-    public Optional<EventListener> getEventListener()
+    public List<EventListener> getEventListeners()
```

Comment:
I think this should be `Optional<List `

Reply:
Thanks @jaystarshot for taking a look. I am not fully sure, I understand that. List can have zero elements, and if we are concerned about not breaking existing users, we added a `default Optional<EventListener>` in QueryRunner interface [Link](https://github.com/prestodb/presto/pull/24755/files#diff-6027ca33c24a27d1b02ac99e9884d2ad2b1ec532b4d35118c9f3f8e2b47d54f7R64).